crates/router/src/consts.rs

```diff
@@ -11,6 +11,8 @@ pub(crate) const ALPHABETS: [char; 62] = [
 ];
 /// API client request timeout (in seconds)
 pub const REQUEST_TIME_OUT: u64 = 30;
+pub const REQUEST_TIMEOUT_ERROR_CODE: &str = "TIMEOUT";
+pub const REQUEST_TIMEOUT_ERROR_MESSAGE: &str = "Upstream did not respond in specified time";
```

Comment:
For the sake of clarity, can we consider saying `connector` instead of `upstream`?